index.js

```diff
@@ -53,6 +54,7 @@ class ShopifyToken {
     this.sharedSecret = options.sharedSecret;
     this.redirectUri = options.redirectUri;
     this.apiKey = options.apiKey;
+    this.access_mode = options.access_mode || '';
```

Comment:
Can we add this as a new parameter to `generateAuthUrl()` instead? If I'm not wrong, in this way it's possible to generate both online and offline access tokens with the same instance.